app/assets/stylesheets/components/_button.scss

```diff
@@ -0,0 +1,25 @@
+.btn {
+  display: flex;
+  width: 264px;
+  padding: 12px 16px;
+  justify-content: center;
+  align-items: center;
+  gap: 8px;
+  align-self: stretch;
+  border-radius: 12px;
+  color: white;
+  // font-family: Inter;
+  font-size: 1em;
+  font-style: normal;
+  font-weight: 300;
+  line-height: normal;
```

Comment:
idem

Reply:
it is done 👍

Reply:
C'est mieux avec les couleurs configurées 👍

Reply:
je suis bien d'accord